apps/arweave/src/ar_serialize.erl

```diff
@@ -375,7 +375,30 @@ nonce_limiter_update_to_binary(2 = _Format, #nonce_limiter_update{
 	CheckpointLen = length(Checkpoints),
```

Comment:
Do you think we can remove the `checkpoints` field from `#nonce_limiter_update`? Now that we have `ar_serialize` query the step_checkpoints_map, I think the `checkpoints` field is redundant, right? (and may int eh future be a source of discrepancy/error)

Not necessary to remove it in this PR, but if that sounds like a good move I can add an issue for later. (whenever "later" actually comes)